rustls/tests/api.rs

```diff
@@ -2422,6 +2422,54 @@ fn test_tls13_exporter() {
     }
 }
 
+#[test]
+fn test_tls13_exporter_maximum_output_length() {
```

Comment:
nb. this test uncovers a panic against rel-0.21. Since nobody has reported it, and I think it's highly unlikely to be reachable except deliberately writing this code (in which case, why not write `panic!()` and save some effort) I don't think it's a pressing concern. However this test could be a backport candidate if we choose to fix that.